src/tokens/transformation/common/leoParser.js

```diff
@@ -45,7 +62,9 @@ module.exports = {
         // Some entries are null in the style-dictionary export.
         if (!entry) continue
         const color = new TinyColor(entry.color).toHex8String()
-        const match = effectColors.find(([key, value]) => value === color)
+        const match =
```

Comment:
Do you mind adding a comment about why we need to do this because I'm not going to remember :laughing: